package.json

```diff
@@ -69,14 +61,20 @@
     "babel-jest": "^23.6.0",
     "babel-plugin-add-module-exports": "^1.0.0",
     "babel-plugin-tester": "^10.1.0",
+    "eslint": "^7.28.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.4.0",
```

Comment:
I think you also want to add `eslint-config-prettier`, and [define prettier as the last plugin in your extends](https://github.com/prettier/eslint-config-prettier#installation).